README.md

```diff
@@ -11,6 +11,8 @@ The superchain configs are made available in minimal form, to embed in OP-Stack
 Full deployment artifacts and genesis-states can be derived from the minimal form
 using the reference [`op-chain-ops`] tooling.
 
+The `semver.yaml` file represents the semantic versioning lockfile for the smart contracts in the superchain.
```

Comment:
Would reword this, since it's a single file, for all superchain targets at the same time.